test/fixtures/python/crypto.invalid.py

```diff
@@ -0,0 +1,259 @@
+## Copyright 2019 VMware, Inc.
+## SPDX-License-Identifier: BSD-2-Clause
```

Comment:
You're also re-licensing code that is licensed as Apache 2

Reply:
Yep, I just used another file.